src/App.js

```diff
@@ -28,20 +50,30 @@ class App extends Component {
     pos.setAttribute('scope', 'row');
     pos.innerHTML = index + 1;
 
+    var name = athlete.competitorName.split(" ");
```

Comment:
`const`  instead of `var`. In this version, we are following properties of immutable resources